.github/workflows/testing.yml

```diff
@@ -24,11 +24,11 @@ jobs:
     steps:
       # Setup the environment.
       - name: Setup Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4.1.0
```

Comment:
Can we do `@v4.1` or something to make sure we get security updates?

Reply:
Done. Thanks for catching that. 

https://github.com/marketplace/actions/setup-go-environment#check-latest-version